site/components/pages/Home/index.tsx

```diff
@@ -366,7 +367,7 @@ export const Home: NextPage<Props> = (props) => {
               </Text>
               <div className="sprouts_col2_textGroup">
                 <Text t="h2" uppercase>
-                  6% WEEKLY YIELD
+                  {props.fiveDayStakingRewards}% WEEKLY STAKING REWARDS
```

Comment:
let's keep it at 'weekly yield', to a first-time reader 'staking rewards' is a nonsense term (frequent flier miles? gold stars?),  and in the text right below we clarify the yield mechanism